modules/otel-send-traces-logs-and-metrics-to-otel-collector-with-sidecar.adoc

```diff
@@ -84,15 +91,23 @@ metadata:
 spec:
   serviceAccount: otel-collector-sidecar
   mode: sidecar
+  mode: sidecar
```

Comment:
Need to remove one of the line mode: sidecar

Reply:
Good catch!